### DISCUSSION TITLE
Possibility to change the frequency of output to the console

### DISCUSSION THREAD

**Discussion by mst-parkhachev** [firemodels/fds#13415] [General]
Dear colleagues!

Please tell me if there is an FDS setting that regulates the frequency of console output of information about the simulated step (see screenshot). Is it possible to output a line every step, or every N seconds of fire calculation?
![photo_2024-09-06_16-55-26](https://github.com/user-attachments/assets/3bdc1c6f-a3ee-4d83-a22e-12cfdc51cf29)

**Comment by mcgratta** [comment]
`DIAGNOSTICS_INTERVAL`
